### DISCUSSION TITLE
Use __future__.annotations

### DISCUSSION THREAD

**Discussion by zrquan** [encode/httpcore#954] [Ideas]
Use the new typing style, just like [in httpx](https://github.com/encode/httpx/pull/3068/files).

**Comment by lovelydinosaur** [comment]
Okay. I do think they're more readable, so you're welcome to escalate this into an issue. Please include an example in the description.

**Reply by karpetrosyan** [reply]
I think it will be also useful to add a CI check, that will enforce us using the new python annotations, otherwise we should periodically update the old ones
